test/netjsongraph.render.test.js

```
@@ -1819,3 +1819,136 @@ describe("mapRender clustering label visibility", () => {
+      nodes: [
+        {
+          id: "node-1",
+          location: {lat: 10, lng: 20},
+        },
+      ],
+      links: [],
+    };
+    const container = document.createElement("div");
+    container.setAttribute("id", "map");
+    document.body.appendChild(container);
+    const graph = new NetJSONGraphCore(data);
```

ESLint

[error] 'graph' is already declared in the upper scope on line 37 column 7.

(no-shadow)